libnetdata/dictionary/dictionary.c

```diff
@@ -147,6 +150,11 @@ void dictionary_register_delete_callback(DICTIONARY *dict, void (*del_callback)(
     dict->del_callback_data = data;
 }
 
+void dictionary_register_merge_callback(DICTIONARY *dict, void (*del_callback)(const char *name, void *old_value, void *new_value, void *data), void *data) {
```

Comment:
minor - should we rename parameter  to `merge_callback` instead of `del_callback` ?

Reply:
renamed to `conflict`